### DISCUSSION TITLE
Multiple instances and ability to specify folders in specific situations

### DISCUSSION THREAD

**Discussion by GorchPortley** [UniSharp/laravel-filemanager#1247] [Q&A]
Hello!

I am using lfm to show a user their own folder to manage project files when creating their project pages, but I also want to use lfm on the project page for their customers to browse the project folder that resides in the user folder.

I will have multiple users with multiple projects each requiring their own browser on the project page.

Is there a way to specify the directory for individual instances of lfm if I embed it into the page?

**Comment by streamtw** [comment]
This can achieved by implement custom logic in `userField()`. See config document here: https://unisharp.github.io/laravel-filemanager/config#private_folder_name .
